torch/nn/parallel/distributed.py

```diff
@@ -630,6 +630,9 @@ def _ddp_init_helper(
             self.find_unused_parameters,
             self.gradient_as_bucket_view,
             param_to_name_mapping,
+            # User can set dist._DEFAULT_FIRST_BUCKET_BYTES to tune DDP first
+            # bucket.
+            dist._DEFAULT_FIRST_BUCKET_BYTES
```

Comment:
nit: since we are passing python config to c++, we may be able to kill this 'module.attr("_DEFAULT_FIRST_BUCKET_BYTES") = ::c10d::kDefaultFirstBucketBytes;' that is used for sharing config btw python and C++, and only define the constant in Python

Reply:
Sounds good, filed https://github.com/pytorch/pytorch/issues/62790 to track it